DEVELOPMENT.md

```diff
@@ -1,5 +1,38 @@
 # Development
 
+## Environment
+
+###kubectl
+This is the basic CLI tool you use to communicate with a k8s cluster. You can use this to query the cluster to see what’s running on it, or make changes to it. 
+
+Whilst you can use kubectl to deploy and manage your application, using YAML descriptor files, it’s a little too low-level, so we’ll use helm to do that for us (see below), but you’ll use kubectl a lot for diagnostics.
+
+To find more information about kubectl and installation see [Kubernetes Documentation](https://kubernetes.io/docs/tasks/tools/).
+
+### Helm
+
+This is another CLI tool which talks to the k8s cluster. It uses YAML template files that describe an entire deployable application, and provides versioning and deployment/undeployment semantics. It also has a whole set of terminology all of its own:
+
+* A chart is package of YAML files that describe a deployable application. You can get charts from a central repository, or you can build your own (which we will be doing). These YAML files describe k8s statefulsets, services and ingresses, and all get deloyed together. Charts are explicitly versioned - the chart itself has a version, and the application that it deploys also has a version.
+* A release is a deployment of a Helm chart to a k8s cluster. You can deploy the same chart multiple times to the same cluster (e.g. for rolling upgrades)
+
+To find more information about Helm installation see [Installing Helm](https://helm.sh/docs/intro/install/).
+
+### Maven
+
+Data 
+Center Helm Chart project build is managed by Apache Maven, therefore maven should be installed as part of developement setup. 
+
+## Prerequisites 
+Before deploy and setup Atlassian's Data Center products the following resources are required:
+* A Kubernetes cluster
+* An Ingress Controller
+* A database
+* Have a `shared-home` volume
+* Setup `local-home` volume
+
+To see how to create and setup those resources follow [Prerequisites Documentation](https://atlassian-labs.github.io/data-center-helm-charts/userguide/PREREQUISITES/).
+
```

Comment:
A lot of this content feels like its duplicated from the PREREQUISITE doc Would it not be better to add this detail to it and then reference it from here?